### ISSUE TITLE
Installation instructions

### ISSUE THREAD

**Issue by oatspear** [eievui5/evunit#28]
Greetings!
I would love to try this out, but I see there are neither installation instructions nor any releases.
I am assuming that, for now, the recommended method is to clone the source and build it with Rust tools, is that right?

**Comment by evie-calico** [comment]
Yeah, to build it you need to [install Rust](https://www.rust-lang.org/tools/install).

Then install evunit like this:
```
git clone https://github.com/eievui5/evunit/
cd evunit
cargo install --path .
```

Assuming rust is properly configured, this *should* put it on your path.

Alternatively, you can just run `cargo build --release` and copy the binary out of `target/release/`

**Comment by evie-calico** [comment]
Actually, this is on crates.io (https://crates.io/crates/evunit), so you only need to run `cargo install evunit`.

**Comment by evie-calico** [comment]
Closed by 28ae1e2
